### DISCUSSION TITLE
tiles do not update

### DISCUSSION THREAD

**Discussion by Geronyy** [spatie/laravel-dashboard#82] [Q&A]
Let's take a look at the weather tile as an example. I have attached all the relevant code here.

If I check the API call with Postman, it works too. I have the feeling that the "fetch commands" are not working.

Can you help?

---------------------------------------------------
app/Console/Kernel.php

    /**
     * Define the application's command schedule.
     *
     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
     * @return void
     */
    protected function schedule(Schedule $schedule)
    {
        // $schedule->command('inspire')->hourly();
        $schedule->command(\spatie\TimeWeatherTile\Commands\FetchOpenWeatherMapDataCommand::class)->everyMinute();
        $schedule->command(\Spatie\TimeWeatherTile\Commands\FetchBuienradarForecastsCommand::class)->everyMinute();

        $schedule->command(\VineVax\UptimeRobotTile\Commands\FetchUptimeRobotDataCommand::class)->everyMinute();

        $schedule->command(\VitorHugoRo\StockTile\Commands\FetchStockDataCommand::class)->everyMinute();
    }

---------------------------------------------------
config/dashboard.php

'time_weather' => [
            'open_weather_map_key' => env('294c5fdb2d2d*********4c8e8eb4395cef'),
            'open_weather_map_city' => 'Lindau',
            'units' => 'metric', // 'metric' or 'imperial' (metric is default)
            'buienradar_latitude' => env('47.55'),
            'buienradar_longitude' => env('9.6833'),
        ],

---------------------------------------------------
config/dashboard.php

**Comment by Geronyy** [comment]
also there is no fetched data in the database table...

<img width="719" alt="Bildschirmfoto 2021-09-15 um 18 54 59" src="https://user-images.githubusercontent.com/48752656/133476501-38b2a890-8904-4f20-8080-483addffeb1a.png">

**Reply by freekmurze** [reply]
It seems to work for us, and we currently have no time to take a look. Try to debug it yourself, if our package is in error, I'd appreciate a PR. Good luck!
